api/eds.proto

```diff
@@ -186,10 +196,6 @@ message LoadStatsRequest {
 message ClusterLoadAssignment {
```

Comment:
Can you add the "next available field number" comment above? Would be good to start tracking this when we delete fields.